lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -111,6 +111,31 @@ function findDifference(path, actual, expected) {
     const keyPath = path + keyAccessor;
     const expectedValue = expected[key];
 
+    if (key === '_includes') {
+      if (!Array.isArray(expectedValue)) throw new Error('Array subset must be array');
+      if (!Array.isArray(actual)) {
+        return {
+          path,
+          actual: 'Actual value is not an array',
+          expected,
+        };
+      }
+
+      for (const expectedEntry of expectedValue) {
+        if (actual.find(actualEntry => !findDifference(keyPath, actualEntry, expectedEntry))) {
```

Comment:
I think it will be very unexpected and potentially buggy to allow an item to match more than once. Can you make it exclusionary?

Reply:
There may be a better name to match this behavior too... maybe `_subset` or `_subarray`?

Reply:
Ah, and there is a README to update too fyi